src/introspection.ts

```diff
@@ -221,11 +221,16 @@ type mapIntrospection<
     ? Query['__schema']['subscriptionType']['name']
     : never;
   types: mapIntrospectionTypes<Query, Scalars>;
+  introspection: Query;
```

Comment:
Why are we adding the full introspection here again?

Reply:
just to clarify why we're hesitant about `introspection` and `typesList` here. This makes our future plans for #10 complex.

The `mapIntrospection<>` output isn't just an intermediary type to us, but a “format” that we wish to output directly (either in the LSP or in API/schema builders) in the future. So, adding more formats and variations to it will make things difficult.